.circleci/config.yml

```diff
@@ -33,6 +33,18 @@ commands:
           fail_only: true
           only_for_branches: main
           webhook: '${SLACK_WEBHOOK}'
+  install_yq:
+    description: "Install yq"
+    steps:
+      - run:
+          name: Install pip3
+          command: |
+            sudo apt-get update || sudo apt-get update
+            sudo apt-get install python3-pip
+      - run:
+          name: Install yq with pip3
+          command: |
+            pip3 install yq
```

Comment:
Is this enough? I mean as far as I know `yq` depends on `jq`. Don't you have install that as well? 🤔

Reply:
Good catch! However, in the CI, it's enough (it was green).